src/app.js

```diff
@@ -1 +1,42 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const inputArray = minimist(process.argv.slice(2))._;
+
+const add = () => {
+  let total = 0;
+
+  for (let i = 1; i < inputArray.length; i++) {
+    total += inputArray[i];
+  }
+
+  return total;
```

Comment:
`.reduce` feels very relevant here
same for multiply
```suggestion
  return args.reduce(( ... ) => { ... }, 0);
```